addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -32,7 +32,7 @@ class Im_LivechatReportChannel(models.Model):
         string="Day of the Week",
         readonly=True,
     )
-    time_to_answer = fields.Float("Response Time (sec)", digits=(16, 2), readonly=True, aggregator="avg", help="Average time in seconds to give the first answer to the visitor")
+    time_to_answer = fields.Float('Response Time', digits=(16, 6), readonly=True, aggregator="avg", help="Average time in seconds to give the first answer to the visitor")
```

Comment:
Help is wrong now that it's computed in minutes. Also, we try to use double quotes :) You might want to setup ruff to avoid format issues :)